File: Back-End/Entidades/Multa.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace Entidades
{
    public class Multa
    {
        public int Id { get; set; }
        [StringLength(100,ErrorMessage = "La descripción de la multa no debe superar los 100 caracteres.")]
        public string Descripcion { get; set; }
        public byte PuntosMenos { get; set; }

        public List<MultaAutoAutomivilista> MultasAutoAutomivilistas { get; set; }
    }
}

File: Front-End/src/app/Modelos/multa.ts
export class Multa {
    public id : number;
    public descripcion : string;
    public puntosMenos : number;

    public constructor( data: any = {}) {
        this.id = data.id || null ;
        this.descripcion = data.descripcion || '' ;
        this.puntosMenos = data.puntosMenos || 0 ;
    } 
}
